sdk/include/psdk/shellapi.h

```diff
@@ -281,12 +279,19 @@ typedef struct _NOTIFYICONDATAW {
 #endif
 } NOTIFYICONDATAW,*PNOTIFYICONDATAW;
 
+/* NOTIFYICONDATA sizes to force older than current structs */
+#if (NTDDI_VERSION >= NTDDI_WIN2K)
```

Comment:
I would avoid putting an NTDDI_VERSION there since this is the size of the structure up and including Win2000, according to its definition given above.

Reply:
No: this is NT4- size, NT5 size is much bigger.